integration_tests/e2e/goal/addNote.cy.ts

```diff
@@ -6,7 +6,7 @@ import AddNotePage from '../../pages/goal/AddNotePage'
 context('Add a note', () => {
   beforeEach(() => {
     cy.task('reset')
-    cy.task('stubSignIn')
+    cy.task('stubSignInAsUserWithEditAuthority')
```

Comment:
The cypress task `stubSignIn` is a logged in user, but with no roles at all (see the changes further down in `auth.ts`)